uaclient/messages/__init__.py

```diff
@@ -1204,6 +1204,160 @@ class TxtColor:
+)
+CLI_VULNERABILITY_DATE_OUTDATED = t.gettext(
+    """\
+The vulnerabilities data used in the system is outdated by {t_diff} days/hours
+To update the data please run with --update:
+
+    $ {cmd} --update
+"""
+)
+
+CLI_VULNERABILITY_SHOW = t.gettext("show information about a vulnerability")
+CLI_VULNERABILITY_SHOW_DESC = t.gettext(
+    """\
+Show all available information about a given security issue.
+"""
+)
+CLI_VULNERABILITY_SHOW_ISSUE = t.gettext(
+    "Security vulnerability ID to display information."
+    " Format: CVE-yyyy-nnnn, CVE-yyyy-nnnnnnn or USN-nnnn-dd"
+)
+CLI_VULNERABILITY_SHOW_NOT_AFFECTED = t.gettext(
+    "{issue} does not affect your system"
+)
+CLI_VULNERABILITY_SHOW_PUBLIC_URL = t.gettext("Public URL: {}")
+CLI_VULNERABILITY_SHOW_PUBLISHED_AT = t.gettext("Published at: {}")
+CLI_VULNERABILITY_SHOW_JSON_PUBLISHED_AT = t.gettext(
+    "Ubuntu vulnerability data published at: {}"
+)
+CLI_VULNERABILITY_SHOW_APT_UPDATED_AT = t.gettext(
+    "APT package information updated at: {}"
+)
+CLI_VULNERABILITY_SHOW_APT_UPDATED_AT = t.gettext(
+    "APT package information updated at: {}"
+)
+CLI_VULNERABILITY_SHOW_UBUNTU_PRIORITY = t.gettext("Ubuntu priority: {}")
+CLI_VULNERABILITY_SHOW_CVSS_SCORE = t.gettext("CVSS score: {}")
+CLI_VULNERABILITY_SHOW_CVSS_SEVERITY = t.gettext("CVSS severity: {}")
+CLI_VULNERABILITY_SHOW_DESCRIPTION = t.gettext("[DESCRIPTION]\n{}")
+CLI_VULNERABILITY_SHOW_NOTES = t.gettext("[NOTES]\n")
+CLI_VULNERABILITY_SHOW_RELATED_USNS = t.gettext("[RELATED USNs]\n")
+CLI_VULNERABILITY_SHOW_RELATED_CVES = t.gettext("[RELATED CVEs]\n")
+CLI_VULNERABILITY_SHOW_AFFECTED_PKGS = t.gettext(
+    "[AFFECTED INSTALLED PACKAGES]\n"
+)
+
+CLI_VULNERABILITY_UPDATE = t.gettext(
+    "update the vulnerability data in your machine"
+)
+CLI_VULNERABILITY_UPDATE_DESC = t.gettext(
+    """\
+Updates the vulnerability data stored in the machine.
+If no vulnerability data exists yet, the command will
+download it
+"""
+)
+CLI_VULNERABILITY_UPDATE_CHECK_NEW_DATA = t.gettext(
+    """\
+Checking for the availability of newer vulnerability data.
+Please wait a moment while we verify if a new version is ready for download.
+"""
+)
+CLI_VULNERABILITY_UPDATE_NEW_DATA_FOUND = t.gettext(
+    """\
+Found a newer version of the data.
+Fetching and processing the new data…
+"""
+)
+CLI_VULNERABILITY_UPDATE_NO_NEW_DATA_FOUND = OKGREEN_CHECK + t.gettext(
+    """\
+ Vulnerability data in the system is already up-to-date
+"""
+)
+CLI_VULNERABILITY_UPDATE_DATA_UPDATED = OKGREEN_CHECK + t.gettext(
+    """\
+ Vulnerability data in the system is now up-to-date
+"""
+)
+
+CLI_VULNERABILITY_LIST = t.gettext(
+    "list the vulnerabilities that affect the system"
+)
+CLI_VULNERABILITY_LIST_DESC = t.gettext(
+    """\
+    List the fixable vulnerabilities that affects the system.
+    By default, the command will only list CVEs. To display
+    USNs instead, run the command with the --usns flag."""
```

Comment:
You need to either `dedent` this or put it at the start of the line (as the other multiline examples). Otherwise (as you are setting `preserve_description`) you get:
```
# pro vulnerability list --help
usage: pro vulnerability list [-h] [--data-file DATA_FILE] [--all] [--usns]
                              [--unfixable] [--manifest-file MANIFEST_FILE]
                              [--series SERIES] [--update]

    List the fixable vulnerabilities that affects the system.
    By default, the command will only list CVEs. To display
    USNs instead, run the command with the --usns flag.

options:
(...)
```

Which is not wrong per se but looks weird, and is different from the rest of the descriptions.

Reply:
I see `--usns` in the options here, but in this case I think the duplicate is ok in the description as it targets a common use case, and this option is 'more important' than the other ones.